dev/deploy.rst

```diff
@@ -22,6 +22,62 @@ languages!
 
 .. _manual_code_deploy:
 
+Deploy Artifacts
+----------------
+
+During the deploy process, robotpy will attempt to generate a ``deploy.json`` that can provide
```

Comment:
Adding deploy.json can't fail, so remove 'attempt'

Reply:
Changed the wording here